spdk.spec

```
@@ -8,7 +8,7 @@
 %bcond_with doc
 
 Name: spdk
-Version: 20.01.1
+Version: 21.01-rc1
```

```suggestion
Version: 21.01~rc1
```

Separate pre-release tags with a tilde.